README.md

```diff
@@ -119,71 +119,177 @@ function soma(x: number, y: number) {
 
 Adicionando `:number` na frente das variáveis o problema da concatenação não desejada já seria facilmente eliminado.
 
+<div align='center'>
+  <img src='./.github/images/soma.gif'/>
+</div>
+
+---
+
+Mas você já pensou em tipar uma função? Bem, não é exatamente isso, mas em TypeScript é possível tipar o retorno de uma função!
+
+Isso permite ter a segurança que uma função irá retornar determinado tipo, por exemplo:
+
+```ts
+const somar10 = (n: number): number => n + 10;
+
+function resposta(): string {
+  return 'sim!';
+}
+```
+
+Como demonstrado acima, tipar um retorno de função é parecido com tipar uma variável: apeanas coloque um `:tipo`, porém depois dos parênteses.
+
+Caso o tipo retornado não correnponda com o tipo do retorno:
```

Comment:
Gostei bastante das suas adições! Se não for pedir muito pode corrigir esses erros ortográficos que eu marquei? hehe